docs/_docs/integration-checks-reference.md

```diff
@@ -229,10 +229,12 @@ and return:
 
 Typically you could return the first object from the `results` array as part of a
```

Comment:
I think this whole paragraph is kind of weird? Why would you only return the first object? Most Triggers return all the objects, then we dedup? Not sure why we even have this text in this check - seems like we could focus on just the text you are working on.

Reply:
I thought this was a nice example of another way the mismatch could manifest and how to handle it - it just doesn't explain in detail why it's a problem, which is what I was trying to do. It does raise a question that I was confused about anyway, though - we don't dedupe on hook triggers (as far as I know - am I wrong on that?), but do we on the poll results from performList? In any case, I think you're right that only returning the first result is an unnecessary note since we're focused on the shape of the data.